### PR TITLE
Dashboard page pom/features/sd full implementation of working tests

### PR DESCRIPTION
**Pull request by Bryar-Frank**

### CODE REVIEW

File: cucumber-selenium-tests/src/test/resources/com/skillstorm/dashbord.feature
Comment (Anthowu07) [inline cucumber-selenium-tests/src/test/resources/com/skillstorm/dashbord.feature:1]: small typo in filename

File: cucumber-selenium-tests/src/test/java/com/skillstorm/PageObjects/DashboardPage.java [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+import com.skillstorm.PageObjects.Components.Navbar.DashboardNavbar;
+import com.skillstorm.PageObjects.Components.Navbar.LandingNavbar;
+import com.skillstorm.PageObjects.Components.Navbar.Navbar;
+import com.skillstorm.PageObjects.Interfaces.Component;
+
+public class DashboardPage extends Page {
+//#region Static fields
+
+    //Names
+    public static final String CMP_LANDING_NAVBAR_NAME = "Landing Navbar";
```
Comment (achuggi1) [inline cucumber-selenium-tests/src/test/java/com/skillstorm/PageObjects/DashboardPage.java]: Low impact: Different naming conventions in same file.

File: cucumber-selenium-tests/src/test/java/com/skillstorm/PageObjects/DashboardPage.java [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+    }
+
+    ////////// UNIQUE METHODS ////////////////////
+
+    /**
+     * Retrieves the table name associated with a button name.
+     * @param btnName - name of the button to retrieve the table name for.
+     * @return - name of the table associated with the button.
+     */
+    public String getTableNameFromButtonName(String btnName) {
```
Comment (achuggi1) [inline cucumber-selenium-tests/src/test/java/com/skillstorm/PageObjects/DashboardPage.java]: Question: button names have typically been tied to Gherkin files, but the table name returned is the React ID of the table tag. Is this meant to get the table name tied to the Gherkin file or is this correct?

**Review by achuggi1 — COMMENTED** [review]
Low impact and question comments in the changes.

File: cucumber-selenium-tests/src/test/java/com/skillstorm/PageObjects/DashboardPage.java (hunk 2 above)
Comment (Bryar-Frank) [reply cucumber-selenium-tests/src/test/java/com/skillstorm/PageObjects/DashboardPage.java]: haven't played around with this for a while, and the naming system i used to start with was terrible, fixing it now

File: cucumber-selenium-tests/src/test/resources/com/skillstorm/dashbord.feature
Comment (Bryar-Frank) [reply cucumber-selenium-tests/src/test/resources/com/skillstorm/dashbord.feature:1]: fixed

File: cucumber-selenium-tests/src/test/java/com/skillstorm/PageObjects/DashboardPage.java (hunk 1 above)
Comment (Bryar-Frank) [reply cucumber-selenium-tests/src/test/java/com/skillstorm/PageObjects/DashboardPage.java]: fixed

File: cucumber-selenium-tests/src/test/java/com/skillstorm/PageObjects/DashboardPage.java (hunk 2 above)
Comment (Bryar-Frank) [reply cucumber-selenium-tests/src/test/java/com/skillstorm/PageObjects/DashboardPage.java]: modified to have better naming convention and to do what I wanted it to do in the first place. Thanks for the catch!

**Review by achuggi1 — DISMISSED** [review]
LGTM

File: cucumber-selenium-tests/src/test/java/com/skillstorm/StepDefinitions/SDDashboard.java [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+     * 
+     * | objects           | option        | list             |
+     * | Checking Accounts | Checkings     | Checkings List   |
+     * | Savings Accounts  | Savings       | Savings List     |
+     * | Credit Cards      | Credit Cards  | Credit Card List |
+     * | Investments       | Investments   | Investments List |
+     * 
+     */
+    @Given("I have multiple {string}")
+    public void iHaveMulitple(String ojbects) {
```
Comment (msabas1) [inline cucumber-selenium-tests/src/test/java/com/skillstorm/StepDefinitions/SDDashboard.java]: Minor typo --> iHaveMultiple

**Review by msabas1 — DISMISSED** [review]
LGTM, just had one minor comment

File: cucumber-selenium-tests/src/test/java/com/skillstorm/StepDefinitions/SDDashboard.java (hunk 3 above)
Comment (Bryar-Frank) [reply cucumber-selenium-tests/src/test/java/com/skillstorm/StepDefinitions/SDDashboard.java]: i read mulituple out loud haha

**Review by achuggi1 — APPROVED** [review]
LGTM

**Review by Anthowu07 — APPROVED** [review]
Looks good!

**Review by msabas1 — APPROVED** [review]

**Pull request merged**